### DISCUSSION TITLE
Style Settings for a Couple Things

### DISCUSSION THREAD

**Discussion by ChuckWheat** [SlRvb/Obsidian--ITS-Theme#47] [Q&A] [answered]
Hi, I am enjoying the theme, and I'm tweaking it to my liking through the Style Settings plugin, but I am not seeing option for a couple of things I would like to change.
I would like to disable the sliding animation that happens when mousing over items in the settings page.
Checking a box in a list changes the font style to bold and color to an accent color. Can I change it back to strikethrough like the default theme?
Do you have a suggestion as to how I could change these?
Thanks for developing this theme! I'm really liking all the customization options because I don't know CSS (yet) :)

**Comment by SlRvb** [comment]
These are things I'll add to the style settings 👍🏽

**Reply by ChuckWheat** [reply]
Thank you! They work perfectly :)

**Comment by SlRvb** [comment, marked as answer]
Alright, I've added settings for it in the new update: 7785c5c

Should find it here:
- disable the sliding animation: `Workspace > Disable Settings Animation`
- change it back to strikethrough: `Note Elements > Lists > Restore Crossed Checkbox`
